packages/react/src/Button/Button.examples.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import {Button} from '.'
+import {DownloadIcon} from '@primer/octicons-react'
+import {VisuallyHidden} from '../internal/components/VisuallyHidden'
+
+const meta: Meta<typeof Button> = {
+  title: 'Components/Button/Examples',
+} as Meta<typeof Button>
+
+export default meta
+
+export const LoadingStatusAnnouncementSuccessful = () => {
```

Comment:
In these examples, when the loading state is activated, the focus gets lost and removed from the button. I believe this is a [2.4.3](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html) issue, and may result in some screen reader users ending up at the top of the page. Could we avoid focus loss?

Reply:
Good catch. I'll take a look.